src/group/components/GroupView.js

```diff
@@ -124,15 +137,7 @@ const GroupView = () => {
         }}
       >
         <PageHeader header={group.name} />
-        <Restricted permission="group.change" resource={group}>
```

Comment:
Isn't this removing the edit button in Group view? I cannot see the edit button on groups where I'm a manager. Only the share button.

Reply:
It was moved to the group about card, per Hana.